app/views/session_timeout/_warning.html.erb

```diff
@@ -18,6 +18,7 @@
                      class: 'usa-button usa-button--big usa-button--full-width margin-bottom-2' %>
       <%= link_to modal.sign_out,
                   destroy_user_session_path,
+                  role: 'link',
```

Comment:
Was there any change in functionality after adding this role? `link_to` renders an anchor tag and would have the role explicitly stated on it

Reply:
Good catch, it got left in from when I was experimenting with turning it into a button tag.